code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -70,14 +70,6 @@ Difficulty: Hard
 	small_sprite_type = /datum/action/small_sprite/megafauna/bubblegum
 	music_component = /datum/component/music_player/battle
 	music_path = /datum/music/sourced/battle/bubblegum
-
-/mob/living/simple_animal/hostile/megafauna/bubblegum/Initialize(mapload)
```

Comment:
What's this being removed for? There should only have one.

Reply:
Can't uh spawn another on the station

Reply:
this is fine tho, since the sm cascade will trigger a shuttle call anyway

Reply:
And for people want to debug multiple bubblegums

Reply:
also it already had another code that check for single bubblegum spawn in CaveGenerator.dm line 162 `if(ispath(picked_mob, /mob/living/simple_animal/hostile/megafauna/bubblegum)) //there can be only one bubblegum, so don't waste spawns on it`